src/Exception/ApplicationException.php

```diff
@@ -2,13 +2,13 @@
 
 namespace Keboola\DockerBundle\Exception;
 
-class ApplicationException extends \Exception
+class ApplicationException extends \Keboola\Syrup\Exception\ApplicationException
```

Comment:
Tím se ale zase přenáší závislost. Nešlo by konverzi na jinou třídu udělat třeba v Executoru / Controlleru? Nebo to předělat na nějaký interface?